README.md

```diff
@@ -1,23 +1,20 @@
-> This script is unsupported. 
+> This script is not officially supported by LaunchDarkly.
 >
-> To sync individual flags, please use the in-application Compare and Copy functionality instead:
-> https://launchdarkly.com/blog/launched-compare-and-copy-flag-settings-across-environments/
+> To sync individual flags, use the in-application [compare and copy](https://docs.launchdarkly.com/home/code/flag-compare-copy#comparing-and-copying-flag-settings-between-two-environments) functionality instead.
 
 # LaunchDarkly Environment Synchronizer
 
-This Node script simplifies the task of synchronizing flag rollout rules between two different environments. 
-It assumes that both environments are in the same project and account, and takes as input the API keys for 
-the two environments you're syncing.
+This Node script simplifies the task of synchronizing flag rollout rules between two different environments. It assumes that both environments are in the same project and account, and takes as input the API keys for the two environments you're syncing.
 
-The following properties will be synchronized:
+The following properties of each flag are synchronized:
 
-* If the evaluation rules are enabled
-* The archived bit
-* User target rules
-* Attribute targeting rules
-  * To omit clauses that reference segments, which are not necessarily shared between environments, use the `--omit-segments` flag
-* Prerequisites
-* Fallthrough rule
+* Whether the flag is toggled **On** or **Off**, that is, whether the evaluation rules are enabled
```

Comment:
I think it's correct if we use "enabling targeting rules" because the UI explicitly calls each of these targeting. (I'm less certain on "evaluation" because I think technically we have to evaluate, in the SDK sense, to determine if the flag is off.) Updating.